tests/test_ch_line_ending.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+
+import unittest
+import filecmp
+import os
+from SJARACNe.bin.ch_line_ending import ch_line_ending as ch
```

Comment:
I had the exact same problem when trying to use Travis CI with the project. Their server would not recognize SJARACNe folder as a module either. I found that it was because test.py would not check the proper root directory, and would consider ./tests/ as the root and not find SJARACNe folder there. My workaround for this when using Travis CI was to create a conftest.py file in the main directory since pytest uses it to decide the rootdir.

Reply:
I see. That makes sense. Thanks for figuring out a solution.